pkg/keystore/keystore.go

```diff
@@ -106,6 +106,15 @@ func checkSignature(ks *Keystore, prefix string, signed, signature io.ReadSeeker
 	return entities, err
 }
 
+// DeleteAllTrustedKeysForPrefix deletes all trusted keys for the given prefix
+func (ks *Keystore) DeleteAllTrustedKeysForPrefix(prefix string) error {
+	acidentifier, err := types.NewACIdentifier(prefix)
+	if err != nil {
+		return err
+	}
+	return os.RemoveAll(path.Join(ks.LocalPrefixPath, acidentifier.String()))
```

Comment:
We should not delete empty files in /etc because **deleting** them in /etc might **enable** the other keys in /usr (counter-intuitively).